lib/ohai/version.rb

```diff
@@ -19,5 +19,5 @@
 
 module Ohai
   OHAI_ROOT = File.expand_path(__dir__)
-  VERSION = "19.0.6"
+  VERSION = "0.1.0"
```

Comment:
You cannot backrev the version of Ohai, you will break the world.

Reply:
I will revert back the version changes, they are not needed anymore. Thanks